Makefile

```diff
@@ -386,7 +386,7 @@ mo-files: $(MOFILES)
 
 extract:
 	$(RM) $(POTFILE)
-	xgettext -c/ -k_ -k_s -ktranslatable -o po/cpp.pot *.cpp src/*.cpp rss/*.cpp
+	xgettext -c/ -k_ -k_s -ktranslatable --add-comments --language=C++ -o po/cpp.pot *.cpp src/*.cpp rss/*.cpp
```

Comment:
`--add-comments` here overrides previous `-c/` (== `--add-comments=/`), and so `xgettext` extracts a few comments that are not meant for translators.

`-c/` extracts comments that start with `///`, but that sequence is also commonly used for doc-comments. I suggest we introduce a new marker for i18n comments, e.g. `//i18n:`, and run `xgettext` with `--add-comments=i18n:` instead of `-c/`. That way such comments can be easily recognized as notes to translators, and `xgettext` doesn't run the risk of extracting more comments than it should.

Unfortunately `xtr` doesn't support `--add-comments` yet. I filed https://github.com/woboq/tr/issues/16 about this.

I realize I'm asking for quite a bit of work here, so if you don't have time/energy/inclination, please extract this comment into an issue and do the minimal edit instead: remove `--add-comments` here and change your comment in itemviewformaction.cpp to `///`.

Reply:
Good idea, done.
It looks like there were not many `///` comments extracted by xgettext.
I think I got them all (found them by running `make extract` and looking at the `po/newsboat.pot` diff)